source/captiveportal/configuring-a-pre-authentication-redirect-for-captive-portal-users.rst

```diff
@@ -1,43 +0,0 @@
-Configuring a Pre-authentication Redirect for Captive Portal Users
```

Comment:
Why remove this? The option is still there, and worked last I tried it. This is still useful information.

Reply:
You are right, I should not have removed this.

I re-added the file and updated its content here : https://deploy-preview-90--pfsense-docs.netlify.com/captiveportal/captive-portal-redirection.html